### ISSUE TITLE
Leading "/" is being removed

### ISSUE THREAD

**Issue by mudassir0909** [hollandben/grunt-cache-bust#137]
I have a file & it's in directory app/views/folder/somefile.html.erb

```html
        <img src="/assets/images/myimage.png">
```

After cache busting it gets converted to 

```html
        <img src="assets/images/myimage.xxxxxxxxx.png">
```

The issue is that, plugin is removing the leading "/" & when I load the page it returns me 404 because that page is not on the root url of by webserver.

my config is as follows:

```js
    cacheBust: {
      options: {
        encoding: 'utf8',
        algorithm: 'md5',
        length: 16,
        baseDir: '../public',
        ignorePatterns: [ 'favicons.html.erb' ]
      },
      production: {
        files: [{
          src: [ '../app/views/**/*.erb', './javascripts/angular/templates/**/*.html' ]
        }]
      }
    },
```

It was working fine before, broke recently

**Comment by hollandben** [comment]
Hi @mudassir0909 - I've just pushed updates to rewriting the way URLs are resolved - try pulling down `v0.5.0` and see if that fixes it

**Comment by hollandben** [comment]
This will definitely be fixed in `v1.0.0`. You can pull down the `v1.0.0-beta` branch and try that out if you'd like

**Comment by mudassir0909** [comment]
Thanks! I'm already using a temporary workaround, will pull once it is fixed in `v1.0.0`
